src/main/java/org/wildfly/swarm/microprofile/fault/tolerance/hystrix/HystrixCommandInterceptor.java

```diff
@@ -82,6 +79,19 @@ public Object interceptCommand(InvocationContext ic) throws Exception {
         }
     }
 
+    private Setter initSetter(Method method) {
+        HystrixCommandProperties.Setter propertiesSetter = HystrixCommandProperties.Setter();
+
+        Timeout timeout = getAnnotation(method, Timeout.class);
+        if (timeout != null) {
+            // TODO: In theory a user might specify a long value
+            propertiesSetter.withExecutionTimeoutInMilliseconds((int) Duration.of(timeout.value(), timeout.unit()).toMillis());
+        }
+
+        return Setter.withGroupKey(HystrixCommandGroupKey.Factory.asKey("DefaultCommandGroup"))
+                // Each method has a unique command key
+                .andCommandKey(HystrixCommandKey.Factory.asKey(method.toString())).andCommandPropertiesDefaults(propertiesSetter);
```

Comment:
Yep, you're right, I thought `Method.toString()` includes declaring class which is not the case. I'm going to add declaring class.